common.js

```diff
@@ -2,25 +2,25 @@
 
 const submodules = [
   'auth', // Validation of data for authentication/authorization
```

Comment:
Shouldn't `auth` be after `array` if that list is sorted alphabetically?

Reply:
Yes, sorry.